src/Generating/Trees.cpp

```diff
@@ -1299,26 +1300,41 @@ void GetLargeJungleTreeImage(Vector3i a_BlockPos, cNoise & a_Noise, int a_Seq, s
 		a_OtherBlocks.push_back(sSetBlock(a_BlockPos.addedY(-i).addedXZ(1, 1), E_BLOCK_DIRT, E_META_DIRT_NORMAL));
 	}
 
-	int numBranches = (a_Noise.IntNoise2DInt(a_BlockPos.x * a_Seq, a_BlockPos.z * a_Seq) / 11) % 3 + 1;
-	int branchStartHeight = 8 + Height % 11;
-	int branchInterval = (Height - branchStartHeight) / numBranches;
-	for (int i = branchStartHeight; i < (Height - 6); i += branchInterval)
+	int NumBranches = std::max(
+		(a_Noise.IntNoise2DInt(a_BlockPos.x * a_Seq, a_BlockPos.z * a_Seq) / 10) % 4,  // The Original Calculation
+		FloorC(Height / (Height / 2.0f))                                               // Just to assure that no massive trees spawn with just one branch
```

Comment:
Isn't this just 2?

Reply:
yes

i changed it that for every 10 blocks of height at least one branch is spawned (maximum is at 29) 

0-9 minimum is 0
10-19 minimum is 1
20-29 minimum is 2